connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MySQLIntegrationSuite.scala

```diff
@@ -150,13 +151,25 @@ class MySQLIntegrationSuite extends DockerJDBCIntegrationSuite {
     assert(rows.get(4).isInstanceOf[BigDecimal])
     assert(rows.get(5).isInstanceOf[BigDecimal])
     assert(rows.get(6).isInstanceOf[Double])
+    // Unlike MySQL, MariaDB seems not to distinguish signed and unsigned tinyint(1).
+    val isMaria = jdbcUrl.indexOf("disableMariaDbDriver") == -1
```

Comment:
This illustrates a deviation between MariaDB and MySQL Connector/J. It‘s not caused by removing codes in this PR.